tutorials/models_mapping/plot_1_tensor_models.py

```diff
@@ -0,0 +1,333 @@
+
+
+"""
+Tensor Meshes
+=============
+
+Here we demonstrate various ways that models can be defined and mapped to
+tensor meshes. Some things we consider are:
+
+    - Surface topography
+    - Adding structures of various shape to the model
+    - Parameterized models
+    - Models with 2 or more physical properties
+    
+
+"""
+
+#########################################################################
+# Import modules
+# --------------
+#
+
+from SimPEG import Mesh
+from SimPEG import Utils
+from SimPEG import Maps
```

Comment:
To simplify the transition over to the `simulation` changes, would you mind updating the imports to be 
```python
from SimPEG import Utils as utils
from SimPEG import Maps as maps
```

and use `utils.whatever` and `maps.whatever`

Reply:
Whether or not I do this, I will still need to make changes after the simulation class is merged. I'll just wait until then.

Reply:
okay, I am just trying to help simplify things.